components/main_page/wikis/starcraft2/main_page_layout_data.lua

```diff
@@ -0,0 +1,193 @@
+---
+-- @Liquipedia
+-- wiki=starcraft2
+-- page=Module:MainPageLayout/data
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local CONTENT = {
+	theGame = {
+		heading = 'The Game',
+		body = '{{Liquipedia:The Game}}',
+		padding = true,
+		boxid = 1503,
+	},
+	wantToHelp = {
+		heading = 'Want To Help?',
+		body = '{{Liquipedia:Want_to_help}}',
+		padding = true,
+		boxid = 1504,
+	},
+	transfers = {
+		heading = 'Transfers',
+		body = '{{Transfer List|onlyNotableTransfers=true|limit=15}}\n' ..
+			'<div style{{=}}"display:block; text-align:center; padding:0.5em;">\n' ..
+			'<div style{{=}}"display:inline; float:left; font-style:italic;">\'\'[[#Top|Back to top]]\'\'</div>\n' ..
+			'<div style{{=}}"display:inline; float:right;" class="plainlinks smalledit">' ..
+			'&#91;[[Special:EditPage/Player Transfers/{{Current term}}|edit]]&#93;</div>\n' ..
```

Comment:
not how transfer pages are structured on sc2
this will just lead to an empty page

Reply:
tbh i prefer to have the template not used at all
rather do it in lua :P

--> [9b50665](https://github.com/Liquipedia/Lua-Modules/pull/5451/commits/9b506652fe60f3636975cddef20b99f51ab4046f)

Reply:
likely was just missing this: https://liquipedia.net/starcraft2/index.php?title=User%3AElectricalBoy%2FMainPage&type=revision&diff=2556704&oldid=2555905